tests/common/aws_test_runner.c

```diff
@@ -266,6 +275,107 @@ static void RunTests( void )
     #endif
 }
 /*-----------------------------------------------------------*/
+#if defined( AWS_TEST_RUNNER_ENABLE_CLI_INPUT ) && ( AWS_TEST_RUNNER_ENABLE_CLI_INPUT == 1 )
+
+/* Global flag that represents whether a "start" command is received
+ * on the serial console. */
+    static BaseType_t xReceivedCommand = pdFALSE;
+
+/**
+ * @brief The command handler for the "start" command recognized by the
+ * serial console of the FreeRTOS test runner. On receiving the command,
+ * the test runner begins executing tests on the device.
+ *
+ * @return Returns pdFALSE to indicate completion of execution of tests.
+ */
+    static BaseType_t prvRunTestsCommand( int8_t * pcWriteBuffer,
+                                          size_t xWriteBufferLen,
+                                          const int8_t * pcCommandString )
+    {
+        ( void ) pcWriteBuffer;
+        ( void ) xWriteBufferLen;
+        ( void ) pcCommandString;
+
+        /* Set flag to indicate reception of command from serial console to start */
+        /* executing tests. */
+        xReceivedCommand = pdTRUE;
```

Comment:
An "OK" response back to the peer here will help in debugging a broken CLI connection to the actual test issues.

Reply:
Added the response to acknowledge processing of command.

Reply:
Per offline discussion, removed the response for now as the response would be visible only at the end of the test execution. We can consider adding the response string later if there are architectural changes of running the CLI console in a different task than the test runner.